src/service.c

```diff
@@ -1062,6 +1073,20 @@ static void on_custom_activated (GSimpleAction *pAction G_GNUC_UNUSED, GVariant
     ayatana_common_utils_open_url (sUri);
 }
 
+static void onClose (GSimpleAction *pAction G_GNUC_UNUSED, GVariant *pParam G_GNUC_UNUSED, gpointer pUserData)
+{
+    gboolean bSuspendable = rda_session_can_be_suspended ();
+
+    if (bSuspendable)
```

Comment:
This if-clause needs to be moved further up: If a remote session is suspendable, the "Suspend %s Session" menu item shall be shown. If not, it should be hidden.